addons/note/controllers/note.py

```diff
@@ -24,4 +24,5 @@ def note_new_from_systray(self, note, activity_type_id=None, date_deadline=None)
             if activity_type_id:
                 activity_values['activity_type_id'] = activity_type_id
             request.env['mail.activity'].create(activity_values)
+            # VFE NOTE replace by activity_schedule ?
```

Comment:
To answer: here call to create makes sense as it is linked to an user action creating a reminder. It is not an automatically created activity. So no need to call activity schedule I think.